sources/ae/emirates.hjson

```diff
@@ -0,0 +1,113 @@
+{
+    versions: '>=1'
+    type: http
+    production: true
+    wikidata: 'Q19833031'
+    note: Emirates of UAE
+    data: 'Natural Earth Data'
+    attribution: [
+      {
+        label: {
+          en: Made with NaturalEarth
+        }
+        url: {
+          en: http://www.naturalearthdata.com/about/terms-of-use
+        }
+      }
+      {
+        label: {
+          en: © OpenStreetMap contributors
+        }
+        url: {
+          en: https://www.openstreetmap.org/copyright
+        }
+      }
+      {
+        label: {
+          en: Elastic Maps Service
+        }
+        url: {
+          en: https://www.elastic.co/elastic-maps-service
+        }
+      }
+    ]
+    fieldMapping: [
+        {
+            type: id
+            name: iso_3166_2
+            desc: ISO-3166-2 identifier
+        }
+        {
+            type: property
+            name: label_en
+            desc: Emirate name (English)
+        }
+        {
+            type: property
+            name: label_ar
+            desc: Emirate name (Arabic)
+        }
+    ]
+    query: {}
+    name: 'uae_emirates'
+    legacyIds: [
+      'UAE Emirates'
```

Comment:
Alternatively, `United Arab Emirates emirates` or `Emirates of United Arab Emirates` or just `United Arab Emirates`?  🤷  😅 

UAE is shorter and [fits better in the ToC](https://maps.elastic.co/?manifest=testing#file/uae_emirates). See also [USA States](https://maps.elastic.co/?manifest=testing#file/usa_states). But I can understand if we want to avoid shortening the name. Any thoughts @jsanz @kelastic?

Reply:
@nickpeihl thank you for your help with this.

UAE Emirates (with a capital E) makes sense to me, since it would be similar to USA States: https://maps.elastic.co/#file/usa_states 

For https://maps.elastic.co/?manifest=testing#file/uae_emirates would it please be possible to add an Arabic label for "Neutral Zone"?

Reply:
@kelastic I did some cursory research on these "Neutral Zones" and they appear to be disputed territories with neighboring countries. From the worldview perspective of UAE, these are UAE territories. But they are de facto controlled by other entities. 

Since this layer is only UAE territories, I'm considering merging these "Neutral Zones" into their respective emirates in UAE. We have already established a precedent for merging disputed territories into specific country layers such as the [Kashmir region](https://en.wikipedia.org/wiki/Line_of_Control) in [India States layer](https://maps.elastic.co/#file/india_states_v1) and [Taiwan](https://en.wikipedia.org/wiki/Political_status_of_Taiwan) in the [China Provinces layer](https://maps.elastic.co/#file/china_provinces). To avoid topological inconsistencies, we will still maintain a separate opinionated worldview in the [World Countries](https://maps.elastic.co/#file/world_countries) and [Administrative regions](https://maps.elastic.co/#file/administrative_regions_lvl2) layers.

Do you agree that we could include these disputed territories as part of UAE?

Reply:
@nickpeihl Yes